Sources/Link.swift

```diff
@@ -19,6 +19,11 @@ public struct LinkSys {
 }
 
 
+/** A representation of Linked Resources that a field may point to in your content model.
```

Comment:
Does the content of comment need to start on the new line, or would that also work this way (aka generating docs will not change that here?) Also: Consistency: Where do you want your reference documention comments to start: On the new line or after the start of the initial starting symbol?)